Sources/StreamChatUI/VoiceRecording/Views/LockIndicatorView.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+/// A component used to show the (un)locked state of the recording flow.
+open class LockIndicatorView: _View, ThemeProvider {
+    public var content: Bool = false {
+        didSet { updateContentIfNeeded() }
+    }
+
+    // MARK: - UI Components
+
+    open lazy var horizontalStackView: UIStackView = .init()
+        .withoutAutoresizingMaskConstraints
+
+    open lazy var lockViewContainer: UIStackView = .init()
+        .withoutAutoresizingMaskConstraints
+
+    open lazy var container: UIView = .init()
+        .withoutAutoresizingMaskConstraints
+
+    open lazy var stackView: UIStackView = .init()
+        .withoutAutoresizingMaskConstraints
```

Comment:
Especially these 3 require docs because it is difficult to distinguish them since they are all containers